### PR TITLE
AUT-2841 - add new radio button options into "A problem proving your identity" contact form

### PR DESCRIPTION
**Pull request by di-fabs**
## What

Add two new radio options (one of which includes hint text and a hyperlink) into "A problem proving your identity" contact form.

## How to review
in the contact form page, choose the option "A problem proving your identity" and submit the form.
"A problem proving your identity" form will appears and you will be able to see the 2 new options:
- "You had a problem with your bank or building society details"
- You had a problem with your National insurance number
You can read more about getting or finding your National Insurance number at [www.gov.uk/find-national-insurance-number](https://www.gov.uk/find-national-insurance-number)

Welsh version:
In the same page, add a new query parameter to your url: &lng=cy.
The text will appears in welsh.

## Checklist

- [ ] Performance analyst has been notified of the change.

aut-2841-screenshot-en
aut-2841-screenshot-cy

- [ ] A UCD review has been performed.

- [ ] Any necessary changes to the [acceptance tests](https://github.com/govuk-one-login/authentication-acceptance-tests) have been made.

- [ ] Documentation has been updated to reflect these changes.

### CODE REVIEW

**Comment by dbes-gds** [comment]
Graham will only test in build or staging so let's add a feature switch to this PR.

**Comment by di-fabs** [comment]
> Graham will only test in build or staging so let's add a feature switch to this PR.

I pushed the changes for that.

File: ci/terraform/integration.tfvars [hunk 1]
```diff
@@ -11,6 +11,7 @@ support_authorize_controller  = "1"
 support_2fa_b4_password_reset = "1"
 support_2hr_lockout           = "1"
 support_reauthentication      = "1"
+no_photo_id_contact_forms     = "1"
```
Comment (dbes-gds) [inline ci/terraform/integration.tfvars:14]: please could you leave switched off in integration?

Comment (di-fabs) [reply ci/terraform/integration.tfvars]: done

**Review by gtvj — DISMISSED** [review]

**Review by gtvj — DISMISSED** [review]

**Review by dbes-gds — APPROVED** [review]

**Pull request merged**